src/Solver/time_stepping/TimeCluster.cpp

```diff
@@ -229,6 +229,7 @@ void seissol::time_stepping::TimeCluster::computeSources() {
 
 #ifndef ACL_DEVICE
 void seissol::time_stepping::TimeCluster::computeDynamicRupture( seissol::initializers::Layer&  layerData ) {
+  if (layerData.getNumberOfCells() == 0) return;
   SCOREP_USER_REGION_DEFINE(myRegionHandle)
```

Comment:
Just to make really sure: are we good by placing the SCOREP region define after the skip for 0 cells?

Reply:
I think so. It's abit hard to test because the SCOREP stuff is more-or-less broken currently.
However, if we always match START/STOP region, we should be fine?

Reply:
Yes should be fine. SCOREP is not particularly broken, it is just very cumbersome to get it working...